configurations/performance/latte-perf-regression-latency-650gb-upgrade.yaml

```diff
@@ -0,0 +1,53 @@
+# NOTE: this is wrapper for the following exisitng cassandra-stress based perf config:
+#       test-cases/performance/perf-regression-latency-650gb-upgrade.yaml
+#       Does the same as C-S just using latte benchmarking tool (rust).
+
+prepare_write_cmd:
+  - >-
+    latte run --function=write --tag=latte-prepare-01 --duration=162500000
+    -P replication_factor=3 --threads=7 --concurrency=21 --consistency=ALL
+    -P column_size=128 -P column_count=8 -P row_count=650000000 -P offset=0
+    --sampling=10s --retry-number=10 --retry-interval='500ms,5s' --request-timeout=60
+    data_dir/latte/latte_cs_alike.rn
```

Comment:
this a intermediate configuration, right ? 
the end goal would be to base it on custumer_d1 configuration, right ?

Reply:
This is example of using the rune script [1] we have in SCT in real-case scenario simulating C-S specifics.

We can add any other config, including `custumer_d1`, anytime later.

This or similar one must be in SCT to be a reference for all the requests about replacing C-S with latte or making sure that C-S is/(is not).guilty in some test run problems/symptom...

[1] https://github.com/scylladb/scylla-cluster-tests/blob/master/data_dir/latte/latte_cs_alike.rn